src/main/java/com/google/api/codegen/php/PhpSnippetSetRunner.java

```diff
@@ -48,15 +49,15 @@ public GeneratedResult generate(
             ImmutableMap.<String, Object>of("context", context));
 
     String outputFilename = snippets.generateFilename(element).prettyPrint();
-    PhpContextCommon phpContextCommon = new PhpContextCommon();
+    PhpTypeTable phpTypeTable = new PhpTypeTable();
 
     // TODO don't depend on a cast here
     PhpContext phpContext = (PhpContext) context;
```

Comment:
Hmm... didn't I get a bunch of grief about removing this pattern from the Python code? ;)